packages/Card/Card.jsx

```diff
@@ -71,9 +74,37 @@ const Card = ({ variant, children, fullHeight, spacing, ...rest }) => {
     spacingProps.inset = 4
   }
 
+  const ContainerStyles = imageProps =>
+    imageProps &&
+    imageProps.position &&
+    handleResponsiveStyles({ position: imageProps.position }, ({ position }) => {
+      if (!imageProps) return {}
+      const direction = {
+        left: 'row',
+        right: 'row-reverse',
+        top: 'column',
+        bottom: 'column-reverse',
+        none: 'row',
+      }
+      const styles = {
+        display: 'flex',
+        flexDirection: direction[position],
+        justifyContent: 'space-between',
+        '> img': {
+          display: position === 'none' ? 'none' : 'block',
+          margin: 'auto',
+        },
+      }
+      return styles
+    })
+
+  const StyledDiv = styled.div(ContainerStyles)
   return (
-    <StyledCard {...safeRest(rest)} fullHeight={fullHeight} variant={variant} {...spacingProps}>
-      {children}
+    <StyledCard {...safeRest(rest)} fullHeight={fullHeight} variant={variant}>
+      <StyledDiv {...fullBleedImage}>
```

Comment:
It seems as though we will be returning this `StyledDiv` regardless of whether or not the `fullBleedImage` prop is provided.

Can we instead check for `fullBleedImage` and then return this structure. Otherwise just return the existing `StyledCard` This should eliminate unnecessary DOM elements.

Reply:
Makes sense, change implemented